includes/order-functions.php

```diff
@@ -913,3 +906,29 @@ function dokan_order_csv_export( $orders, $file = null ) {
 
     fclose( $output );
 }
+
+/**
+ * Dokan get seller id by order id
+ *
+ * @param  int order_id
+ *
+ * @return int
+ */
+function dokan_get_seller_id_by_order_id( $id ) {
+    $order = wc_get_order( $id );
+    $items = $order->get_items( 'line_item' );
+
+    // if we have multiple sellers then return 0
+    if ( count( $items ) > 1 ) {
```

Comment:
If a vendor has multiple line items or products then its always return 0

Reply:
If a vendor has one items then you just get author ID from first product id. If has multiple Items then you can only pick first one then get author id from this product id